src/components/frame/SubscribedColoniesList/SubscribedColoniesList.tsx

```diff
@@ -21,9 +21,16 @@ const MSG = defineMessages({
 });
 
 const SubscribedColoniesList = () => {
-  const { user, userLoading } = useAppContext();
+  const { user, userLoading, updateUser }: AppContext = useAppContext();
   const canInteractWithNetwork = useCanInteractWithNetwork();
 
+  /* Ensures colony list is up-to-date post create colony flow. */
+  useEffect(() => {
+    if (updateUser) {
+      updateUser(user?.walletAddress);
+    }
+  }, [user, updateUser]);
+
```

Comment:
Why not call this from inside the saga at the end of creation flow ?

Reply:
@rdig I don't believe I can access React context outside of a component (AppContext in this case), so while I could get the latest user while in the saga, I'm not able to call `updateUser`.

The dapp pattern of updating the apollo cache at the end of the saga won't affect the values in context, which is what both the SubscribedColoniesList and ColonySubscription are using to render the colony avatars and display the `Join this colony` btn respectively. 

So I don't believe we have an alternative?

Reply:
Bah. You're correct. I was thinking about the other context we have, but this is React's. Leave it like it is for now, although it might need to move somewhere inside the subscribe/unsubscribe component once @arrenv  gets around to #49 

For now leave it as it is